smdebug/tensorflow/utils.py

```diff
@@ -195,35 +194,49 @@ def get_original_fetch_ops(fetches):
 """
 
 
-def is_parameter_server_strategy(tf_config: str) -> bool:
+def load_tf_config_json(tf_config: str):
     try:
-        tf_config = json.loads(tf_config)
+        return json.loads(tf_config)
     except (json.JSONDecodeError, TypeError):
-        return False  # Do not break for incorrectly set tf_config
-    return "cluster" in tf_config and "ps" in tf_config["cluster"]
+        # if tf_config is None throws TypeError, so return None from next line
+        return None
 
 
-def is_mirrored_strategy(strat):
-    return isinstance(strat, (tf.distribute.MirroredStrategy, ContribMirroredStrategy))
+def is_parameter_server_strategy(tf_config_json) -> bool:
```

Comment:
Will this param be a string or a Python dict? Would be good to comment or add type annotation. Code points to it being a dict.

Reply:
it's a python dict, will add type annotation